bitnami/rabbitmq/templates/_helpers.tpl

```diff
@@ -221,3 +221,12 @@ rabbitmq: ingress.tls
       - Relay on Helm to create self-signed certificates by setting `ingress.selfSigned=true`
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return true if a TLS credentials secret object should be created
+*/}}
+{{- define "rabbitmq.createTlsSecret" -}}
+{{- if and .Values.auth.tls.enabled .Values.auth.tls.autoGenerated (not .Values.auth.tls.existingSecret) }}
+    {{- true -}}
+{{- end -}}
+{{- end -}}
```

Comment:
This is not true. You're creating the TLS secret under other circumstances too, please review this

Reply:
Hi @juan131,

I'm not entirely sure of what you mean, so I will explain my point of view to make sure we are on the same page.

`rabbitmq.createTlsSecret` determines if a TLS secret containing the autogenerated certs should be created. (Maybe I should update the helper description)

This helper is referenced only once, and it is at `tls-secret.yaml` in the following logic:

```
{{- if and .Values.auth.tls.enabled (not .Values.auth.tls.existingSecret) (not .Values.auth.tls.autoGenerated )}}

Use certificates provided as plain-text in values.yaml

{{- else if (include "rabbitmq.createTlsSecret" . )}}

Create autogenerated certs

{{ -end }}
```

I had my doubts if we should continue to support providing the certificates as plain-text, but decided to keep the backward compatibility.

So, users of RabbitMQ have 3 methods to provide TLS certificates:
- existingSecret
- Plaintext (using `caCertificate`, `serverCertificate` and `serverKey`)
- Autogenerated certificates

Reply:
Macro description says: "Return true if a TLS credentials secret object should be created"

BUT a TLS secret is also created in the "Plaintext" case. So, two possibilities:

- Rename the macro and description so it clearly states that returns true if a TLS credentials secret object should be created **WITH AUTOGENERATED CERTS.**
- Include the "plaintext" possibility in this "if"

Reply:
I decided to apply the second option, and have the `if` also include the plaintext scenario.

I also modified the secret generated, to not have duplicated code.